rules/falco_rules.yaml

```diff
@@ -1537,6 +1537,12 @@
         - [[rancher-bridge], "rancher/network-manager"]
         - [[calico-node], "calico/node"]
         - [[scope], "weaveworks/scope"]
+        - [[system-probe], "datadog/agent"]
+    - name: proc_pname_image_suffix
```

Comment:
This is redundant. And I don't think we should include vendor product here.

Reply:
Thanks @Kaizhe what is wrong with commercial products? If they are commonly used why we should not add them :)

Reply:
Nothing wrong with the commercial products. It's a rule we followed for a while. For example, we don't put Sysdig commercial images in the rules as exception. If you feel we should include commercial products in the exception, feel free to bring it to the channel or the community call for a broader discussion.